packages/cli/templates/cloudflare/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "devDependencies": {
     "@cloudflare/workers-types": "^4.20240405.0",
-    "wrangler": "^3.48.0",
+    "wrangler": "^3.63.2",
     "miniflare": "^3.20231030.4"
```

Comment:
maybe bump mini flare and worker-types as well?

Reply:
removed miniflare, bumped types